lib/util/screen.dart

```diff
@@ -22,6 +22,12 @@ class Screen {
   static bool isPortrait(BuildContext context) =>
       Screen.orientation(context) == Orientation.portrait;
 
+  static bool isLandscape(BuildContext context) =>
+      Screen.orientation(context) == Orientation.landscape;
```

Comment:
I feel like we can just use !isPortrait() but it's no big issue.

Reply:
Or you could go `static bool isLandScape(BuildContext context) => !isPortrait(context);`

Reply:
I don't think your second comment makes any difference also I just found it easier to read that as I kept forgetting the ! or miss reading it